pyproject.toml

```diff
@@ -1,46 +1,52 @@
-[tool.poetry]
+[project]
 name = "magnify"
 version = "0.10.0"
 description = "A microscopy image processing toolkit."
-authors = ["Karl Krauth <karl.krauth@gmail.com>"]
+authors = [
+    {name = "Karl Krauth", email = "karl.krauth@gmail.com"},
+]
 readme = "README.md"
+requires-python = ">=3.9, <3.13"
 
-[tool.poetry.dependencies]
-python = ">=3.9, <3.13"
-opencv-python = [
-    { version = ">=4.0", markers = "sys_platform != 'darwin'" },
-    { version = ">=4.0, !=4.7.0.68", markers = "sys_platform == 'darwin'" }
+dependencies = [
+    "opencv-python>=4.0; sys_platform != 'darwin'",
+    "opencv-python>=4.0,!=4.7.0.68; sys_platform == 'darwin'",
+    "numpy>=1.22.0",
+    "scipy>=1.9.0",
+    "pandas>=2.0.3",
+    "plotly>=5.18.0",
+    "tifffile>=2021.11.2",
+    "dask_image>=2024.5.3",
+    "tqdm>=4.64",
+    "types-tqdm>=4.64",
+    "xarray[io]>=2023.01.0",
+    "dask[complete]>=2022.02.0",
+    "catalogue>=2.0.8",
+    "beautifulsoup4>=4.10.0",
+    "lxml>=5.0.0",
+    "confection>=0.0.4",
+    "scikit-learn>=1.2.0",
+    "numba>=0.58.1",
+    "matplotlib>=3.9.0",
+    "seaborn>=0.13.0",
+    "napari>=0.4.16",
```

Comment:
Let's bump napari to minimum version 0.5.0 since there were a lot of updates.

Reply:
Changed to 0.5.0